### DISCUSSION TITLE
Stable Server ?

### DISCUSSION THREAD

**Discussion by TheHellTower** [matrix-org/dendrite#3215] [General]
Hello,

Is this server `Stable` or far from it ? Since it's stated `Beta` but there is no `Roadmap` saying what is missing to support or even the bugs..
And I'm interested in the `Golang` server I don't want to run a `Python` instance !
 
![opera_sfK0Nc848u](https://github.com/matrix-org/dendrite/assets/119943599/02149699-5631-40c0-9e01-0f1a199600e0)

Best Regards,
TheHellTower

**Comment by bradjonesca** [comment]
A roadmap to GA/stable would be very useful
